.github/workflows/check.yml

```diff
@@ -0,0 +1,61 @@
+---
+
+name: Check Set-Up & Build
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events
+  # but only for the calamari (default) branch.
+  pull_request:
+    branches: [manta]
+  # push:
```

Comment:
Actually since we never should push directly into the manta branch, it won't make too much of a difference. And it will force the person to take a moment and think about why they are pushing directly without a pull request. So might as well enable it.